bin/git-stats-importer

```diff
@@ -52,7 +52,7 @@ myRepo.exec("config user.email", function (err, GIT_EMAIL) {
               , all: true
               , "no-color": true
             }, "log");
-            myRepo.exec(command, function (err, data) {
+            myRepo.exec(command + " --pretty=medium --date=default", function (err, data) {
```

Comment:
@furkanmustafa My bad, I fixed that in [`oargv@1.2.0`](https://github.com/IonicaBizau/node-oargv). Thanks!